packages/quantic/scripts/build/deploy-community.ts

```diff
@@ -2,6 +2,7 @@ import * as fs from 'fs';
 import * as path from 'path';
 import {StepLogger, StepsRunner} from './util/log';
 import * as sfdx from './util/sfdx-commands';
+import {SfdxJWTAuth} from './util/sfdx-commands';
 // eslint-disable-next-line node/no-unpublished-require
 const waitOn = require('wait-on');
```

Comment:
Here, that's the same thing as the no-unpublished-import. do a `npm I -D wait-on` and bob-your-uncle it's fixed.
It's a real easy win so I'd do the boy scout developer principle here on this one :P